### PR TITLE
Update Quote icon alignment

### PR DESCRIPTION
**Pull request by mxdvl**
## Why?

It’s not tight enough.

### Before

image

### After

image

### CODE REVIEW

**Comment by HarryFischer** [comment]
I've spotted a couple small issues in Chromatic – 10% day so we can pick it up on Monday? @mxdvl

**Comment by oliverlloyd** [comment]
Would it be possible to add an explanation for why this styling change needs the html wrapping logic to be refactored? That's quite complex code and changes to it are hard to follow based on the code alone.

**Comment by mxdvl** [comment]
@oliverlloyd I’ve separated the changes out in their own PR #4033, this should be much more readable.

**Review by HarryFischer — APPROVED** [review]
Looks better now - thanks
